core/client/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -248,6 +285,14 @@ public void getStatus() throws Exception {
     return conf;
   }
 
+  private void mockFileSystemContextAndMasterClient() {
+    mMockFileSystemContext = PowerMockito.mock(FileSystemContext.class);
```

Comment:
any reason to use `PowerMockito.mock` rather than `Mockito.mock`? I think we are stepping away from using `PowerMockito` in our unit tests. @aaudiber , is it correct?